java/core/src/main/java/com/redhat/rhn/manager/kickstart/cobbler/CobblerDistroSyncCommand.java

```diff
@@ -130,17 +130,23 @@ public void backsyncKernelOptions() {
                         " empty, but corresponding fields in its cobbler distro (uid {})" +
                         " are populated. Aligning the kickstartable tree with the cobbler" +
                         " distro now.", candidate.getId(), distro.getUid());
-                candidate.setKernelOptions(distro.convertOptionsMap(distro.getKernelOptions().get()));
-                candidate.setKernelOptionsPost(distro.convertOptionsMap(distro.getKernelOptionsPost().get()));
+
+                distro.getKernelOptions().ifPresent(
+                        kernelOptions ->
+                                candidate.setKernelOptions(distro.convertOptionsMap(kernelOptions)));
+                distro.getKernelOptionsPost().ifPresent(
+                        kernelOptionsPost ->
+                                candidate.setKernelOptionsPost(distro.convertOptionsMap(kernelOptionsPost)));
+
```

Comment:
Use `map ()` to chain methods:
```suggestion
                distro.getKernelOptions().map(distro::convertOptionsMap).ifPresent(candidate::setKernelOptions);
                distro.getKernelOptionsPost().map(distro::convertOptionsMap).ifPresent(candidate::setKernelOptionsPost);
```

Reply:
another lesson learned. Fixed